public/index.html

```diff
@@ -29,5 +150,22 @@
       To begin the development, run `npm start` or `yarn start`.
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
-  </body>
-</html>
+    <script>
+    // buttons script
+      const { ipcRenderer } = require('electron');
+
+      function closeFunction() {
+        ipcRenderer.send('close-app');
+      }
+
+      function minimizeFunction() {
+        ipcRenderer.send('minimize-app');
+      }
+
+    // automatically set the version in the title of the app
+    var appVersion = require('electron').remote.app.getVersion();
+    document.getElementById('frameTitleText').innerHTML = 'Elundus Core - V' + appVersion;
```

Comment:
I don't think the appVersion variable is filled/loaded correctly. I did a production build on my machine, installed and ran the app, but the titlebar still displays 0.0.0 as version number.

Reply:
yeah, this code was coming from another app I've made but I didn't tested it yet, it shouldn't be too hard to fix I think

Reply:
Okay so the way I used to get app version was by using electron `remote` module, but it seems like it's deprecated now, I tried using it with the new methods ([@electron/remote](https://github.com/electron/remote)) but it didn't worked. So I removed automatic title for now.